src/test/java/org/candlepin/util/UtilTest.java

```diff
@@ -394,4 +394,26 @@ void encodesSpecialCharacters() {
         }
     }
 
+    @Test
+    public void testReorderItem() {
+        List<String> preexisting = List.of("A", "B", "C");
+        List<String> updated = List.of("A", "C");
+        assertTrue(Util.getReorderedItems(preexisting, updated).isEmpty());
+    }
+
+    @Test
+    public void testReorderItemListDifferentOrder() {
+        List<String> preexisting = List.of("A", "B", "C");
+        List<String> updated = List.of("A", "C", "B");
+        List<String> reordered = Util.getReorderedItems(preexisting, updated);
+        assertFalse(reordered.isEmpty());
+        assertFalse(reordered.contains("A"));
```

Comment:
I think being more specific about asserting what got re-ordered is better (in this case asserting that C and B got re-ordered), since now this test could pass even if only one of C or B was in the reordered list